CONTRIBUTING.md

```diff
@@ -5,6 +5,13 @@
 If you haven't read it, ***read the very important notice about the Javalib
 in the [Developer documentation](./DEVELOPING.md)***.
 
```

Comment:
OK I added a section about imports. I'm not exactly sure what other well-established shortcuts we have, so I just put the one for `java.util`. If we find others we can grow the list.